stock_available_to_promise_release/models/stock_picking_type.py

```diff
@@ -14,6 +14,12 @@ class StockPickingType(models.Model):
         "backorder are unreleased if they come from a route configured "
         "to manually release moves",
     )
+    set_printed_at_release = fields.Boolean(
```

Comment:
This is required in case of backorder at release to ensure that 2 packs are not linked to a single ship.
Can you remind me why we would be interested by having this as an option?

Reply:
we need it since the backorder creation is specified on the route and a picking is linked to many routes... we could also set printed at release only if no route is configured with 'no_backorder_at_release'.